code/modules/projectiles/guns/ballistic/launchers.dm

```diff
@@ -43,11 +43,11 @@
 	casing_ejector = FALSE
 
 /obj/item/gun/ballistic/rocketlauncher
-	name = "rocket propelled grenade launcher"
-	desc = "A reusable rocket propelled grenade launcher."
+	name = "pml-9"
```

Comment:
lowercase looks ugly

Reply:
I agree, uppercase breaks the text article formatter though. It's the reason why every model number identified gun starts with lower case.

Reply:
wouldn't `\improper PML-9` work?

Reply:
yes that would fix it.

Reply:
Oh, so THAT'S what the \improper flag does. I'll do this in a minute

Reply:
Oh, so THAT'S what the \improper flag does. I'll do this in a minute

Reply:
Oh, so THAT'S what the \improper flag does. I'll do this in a minute

Reply:
Oh, so THAT'S what the \improper flag does. I'll do this in a minute

Reply:
What the fuck github